### ISSUE TITLE
NPM Audit Security Issue with handlebars

### ISSUE THREAD

**Issue by cholz** [pillarjs/hbs#185]
$ npm audit

                       === npm audit security report ===

                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance

  Moderate        Denial of Service

  Package         handlebars

  Patched in      >=4.4.5

  Dependency of   hbs

  Path            hbs > handlebars

  More info       https://nodesecurity.io/advisories/1300

found 1 moderate severity vulnerability in 1765404 scanned packages
  1 vulnerability requires manual review. See the full report for details.

Kind regards..

**Comment by dougwilson** [comment]
In progress PR: https://github.com/pillarjs/hbs/pull/184

**Comment by emilchristensen** [comment]
Another security audit issue with Handlebars just showed up. Should be patched in >=4.5.2

```
High          │ Arbitrary Code Execution
Package       │ handlebars
Patched in    │ >=4.5.2
Dependency of │ hbs
Path          │ hbs > handlebars
More info     │ https://npmjs.com/advisories/1316
```
